src/static/helpers/slas.ts

```diff
@@ -98,10 +98,10 @@ export async function authorize(
 
   const options = {
     parameters: {
-      client_id: slasClient.clientConfig.parameters.clientId,
+      client_id: slasClient.clientConfig.parameters?.clientId as string,
```

Comment:
can parameters be undefined? 🤔

Reply:
I fixed this since typescript was complaining:
```
'slasClient.clientConfig.parameters' is possibly 'undefined'.ts(18048)
```

But yeah, I think it can be undefined,`parameters` is an optional property on the underlying type for `slasClient.clientConfig`:
```
export declare class ClientConfig {
    baseUri?: string;
    cacheManager?: ICacheManager;
    headers?: BasicHeaders;
    parameters?: CommonParameters;
    retrySettings?: OperationOptions;
    fetchOptions?: RequestInit;
}
```